cpp/src/stream_compaction/distinct.cu

```diff
@@ -14,19 +14,14 @@
  * limitations under the License.
  */
 
-#include "stream_compaction_common.cuh"
-#include "stream_compaction_common.hpp"
+#include <stream_compaction/stream_compaction_common.cuh>
```

Comment:
Please discard the change here, we are using `""` when including local headers according to https://github.com/rapidsai/cudf/blob/branch-22.08/cpp/docs/DEVELOPER_GUIDE.md#includes
```
Use <> instead of "" unless the header is in the same directory as the source file.
```